moped-editor/src/utils/activityLogFormatters/mopedProjectActivity.js

```diff
@@ -108,20 +96,37 @@ export const formatProjectActivity = (change, lookupList) => {
         },
       ],
     };
-  }
-  // we dont need the lookup table
-  else {
-    // If the update is an object, show just the field name that was updated.
+  } else {
+    // If the update is an object, check first for a null object
     if (typeof changeData.new[changedField] === "object") {
-      return {
-        changeIcon,
-        changeText: [
-          {
-            text: `Changed ${entryMap.fields[changedField]?.label} `,
-            style: null,
-          },
-        ],
-      };
+      // if the new field is null, it is because something was removed
+      if (lookupList[changeData.new[changedField]] == null) {
+        console.log(changeData.old);
```

Comment:
Could you remove this console log before this gets merged? It was helpful when I tested locally though! 🙏

Reply:
yes, thanks for catching that! i always forget to delete my logs